app/models/item.rb

```diff
@@ -11,4 +11,8 @@ def quantity(invoice)
   def status(invoice)
     InvoiceItem.where(item_id: id, invoice_id: invoice).first.status
   end
+
+  def ordered_on_date
+    Invoice.find(self.invoice_id).date_conversion
+  end
```

Comment:
Nice job grabbing those invoices and using the `date_conversion` method to format it!